all-imports.html

```diff
@@ -0,0 +1,12 @@
+<!--
```

Comment:
Should I remove the index.html as well?

Side Note, RE: Testing, What should be the approach for testing both the mixin and behavior in hybrid mode since there are variants? Just taking a stab at it, I created 1.x and 2.0 folders with test/index.html running only the 1.x and test/2.0/index.html running both variants.